examples/display_audio_photo/main/idf_component.yml

```
@@ -1,6 +1,6 @@
 description: BSP Display Audio Photo Example
 dependencies:
-  idf: ">=5.0"
+  idf: ">=4.4"
   esp_jpeg: "*"
   esp-box:
     version: "*"
```

To make sure the component manager resolve a correct version:

```suggestion
    version: ">=2.5.0"
```